### DISCUSSION TITLE
How can we edit the theme

### DISCUSSION THREAD

**Discussion by alan345** [thelovekesh/ghost-v4-on-heroku#6] [Q&A] [answered]
Hello!
Thanks for your repo!

How can we edit the theme as we don't have access to the files in heroku?

**Comment by thelovekesh** [comment, marked as answer]
You can't edit things on Heroku. For a detailed explanation and help ask your question [here](https://ask.codebulbs.com)

**Comment by thinker732** [comment]
you can copy/paste your casper theme or wathever in themes/ directory and remove casper in package.json and package.lock.json 
all the changes you do on the theme before deployement will be save
